From 728bb89effe3b2d7ee3db8f944d2a467221d8664 Mon Sep 17 00:00:00 2001
From: Yijing Wang <wangyijing@huawei.com>
Date: Mon, 28 Apr 2014 19:35:09 +0800
Subject: [PATCH 28/68] usb: remove redundant D0 power state set
References: FATE#315518
Git-Commit: febf2f63c1d8b4e622730f52202b9d0374a6c058
Patch-Mainline: v3.15

Pci_enable_device() will set device power state to D0,
so it's no need to do it again after call pci_enable_device().

Signed-off-by: Yijing Wang <wangyijing@huawei.com>
Acked-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/usb/core/hcd-pci.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/usb/core/hcd-pci.c b/drivers/usb/core/hcd-pci.c
index 36ac1ae..94f3b68 100644
--- a/drivers/usb/core/hcd-pci.c
+++ b/drivers/usb/core/hcd-pci.c
@@ -192,7 +192,6 @@ int usb_hcd_pci_probe(struct pci_dev *dev, const struct pci_device_id *id)
 
 	if (pci_enable_device(dev) < 0)
 		return -ENODEV;
-	dev->current_state = PCI_D0;
 
 	/*
 	 * The xHCI driver has its own irq management
-- 
1.8.4.5

